packages/edit-post/src/components/sidebar/settings-header/style.scss

```diff
@@ -1,29 +1,15 @@
-.components-panel__header.edit-post-sidebar__panel-tabs {
-	justify-content: flex-start;
-	padding-left: 0;
-	padding-right: $grid-unit-05;
-	border-top: 0;
-
-	ul {
-		display: flex;
-	}
-	li {
-		margin: 0;
-	}
-}
-
 .components-button.edit-post-sidebar__panel-tab {
 	border-radius: 0;
-	height: 50px - $border-width;
+	height: $grid-unit-60;
 	background: transparent;
 	border: none;
 	box-shadow: none;
 	cursor: pointer;
 	display: inline-block;
 	padding: 3px 15px; // Use padding to offset the is-active border, this benefits Windows High Contrast mode
 	margin-left: 0;
-	font-weight: 400;
-	color: $dark-gray-900;
+	font-weight: 500;
+	color: $dark-gray-primary;
```

Comment:
Can't we remove the color and rely on the inheritance instead or is this really needed?